### DISCUSSION TITLE
ERROR: Dependency "gobject-introspection-1.0" not found, tried pkgconfig and cmake

### DISCUSSION THREAD

**Discussion by Dolbaeb777K** [Aylur/ags#313] [Q&A]
When installing ABS on the openSUSE tumbleweed distribution, I did not find the instructions for openSUSE. I thought that the right instruction for this is "#clone, build, install". But after entering the "mason setup build" command, I got this error (shown in the screenshot). How can I solve the problem?
![image](https://github.com/Aylur/ags/assets/126266582/a677a3e9-3ab9-4575-9fb6-d89d176d8140)

**Comment by Dolbaeb777K** [comment]
[meson-log.txt](https://github.com/Aylur/ags/files/14358927/meson-log.txt)

**Comment by Aylur** [comment]
you still need to install the dependencies
It is not on the wiki, because I never used openSUSE, and didn't take the time to find them
